tests/test_models/test_engine/test_file_storage.py

```diff
@@ -113,3 +113,53 @@ def test_save(self):
         with open("file.json", "r") as f:
             js = f.read()
         self.assertEqual(json.loads(string), json.loads(js))
+
+    @unittest.skipIf(models.storage_t == 'db', "not testing file storage")
+    def test_get(self):
+        """
+        Test the get() function
+        creates a new object of any class
+        and checks if the object saves corectly
+        """
+        state = State()
+        state.name = "Fictional state"
+        storage = FileStorage()
+        storage.new(state)
+        storage.save()
+        self.assertEqual(storage.get(State, state.id), state)
+
```

Comment:
Can take in account you can also make a chained query and track the performance to the deep